### ISSUE TITLE
Using Collection field of type:date, default: now throws date.toUpperCase error

### ISSUE THREAD

**Issue by bartonhammond** [perak/kitchen-site#87]
The ~/.meteor-kitchen/templates/bootstrap/client_accounts.js, function **Helpers.formatDate** needs else clause, line 86 should be **else**.  
```javascript 
		if(date.toUpperCase() == "NOW") {
			date = new Date();
		} else {                          //<<<<<<<line 86
		    if(date.toUpperCase() == "TODAY") {
			d = new Date();
			date = new Date(d.getFullYear(), d.getMonth(), d.getDate(), 0, 0, 0, 0);
		   }
               }
```
Question:  Are the ~/.meteor-kitchen files available somewhere for pull requests?

**Comment by perak** [comment]
Hi @bartonhammond 

- Thank you for bug report, I'l fix it in next version (v0.9.37).

- Source code: sorry, "meteor kitchen" source code is not yet open (btw it's written in C++).

P.S. maybe interesting to you: v0.9.36 is just uploaded few minutes ago - added support for OAuth logins and Collection2 (experimental).

**Comment by bartonhammond** [comment]
Yeah - I noticed things changed - I am working with Designer and things got
a little wonky (messed up).

As to code being available, I meant the code that is installed with
meteor-kitchen under ~/.meteor-kitchen/templates such as the file I noted
above.

**Comment by perak** [comment]
What's messed up in GUI? 

BTW, I just noticed that for all existing apps with user account system you need to check "login_with_password" inside application object (it's checked by default for new apps, but is left unchecked for existing ones... bugs and more bugs :) ).

There is no repository for kitchen template files. If you find more bugs just report here please until I open the complete source.

Thanks!

**Comment by bartonhammond** [comment]
Oh - nothing is messed up with the GUI now - there was just some
strangeness while I was navigating and  trying to download the json source
- I think the site was reloading while I was working.  No problem now
though.

Got the "login_with_password" - thanks for the heads up.

btw, is there a way to change the Register to have a "validate password"?

**Comment by perak** [comment]
Do you think on validate e-mail? Not yet, I'l add that soon. Currently working on a project for customer using meteor-kitchen, so I am adding features on-the fly as I need them. In next few days there will be a lot of new stuff added into "kitchen" ("validate e-mail" is one of required features).

**Comment by bartonhammond** [comment]
"validate e-mail" is good when registering.  I'm referring to "validate
password" when registering.

**Comment by perak** [comment]
Ah, I understand. Not yet possible. Please add separate issue for that. Thanks.

**Comment by bartonhammond** [comment]
Done: https://github.com/perak/kitchen-site/issues/88

**Comment by perak** [comment]
:+1:
